pipeline/01_parser.py

```
@@ -51,36 +52,37 @@ def find_libreoffice():
+    """opendataloader_pdf로 여러 PDF를 한 번에 Markdown으로 변환.
+    JVM 부팅이 호출당 ~수백ms라 배치가 필수. 반환: dict[절대경로]→markdown 텍스트."""
+    if not file_paths:
+        return {}
+    abs_paths = [os.path.abspath(p) for p in file_paths]
+    results = {p: "" for p in abs_paths}
+    with tempfile.TemporaryDirectory() as tmp:
+        try:
+            opendataloader_pdf.convert(
+                input_path=abs_paths, output_dir=tmp, format="markdown"
+            )
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)